### PR TITLE
add defaultTool to [react-svg-pan-zoom]

### PR DESCRIPTION
**Pull request by tvwenger**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<[url here](https://github.com/chrvadala/react-svg-pan-zoom/blob/main/docs/documentation.md)>>

Fixes https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/57908

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-svg-pan-zoom ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-svg-pan-zoom@latest/))
was missing the following properties:
1. Toolbar
1. Miniature
1. openMiniature
1. closeMiniature
1. Viewer
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e11cb5cf7676ae6f5cd5e2cecc72eefaf7e29b15
</p>

File: types/react-svg-pan-zoom/index.d.ts [hunk 1]
```diff
@@ -58,6 +59,9 @@ export type ToolbarPosition = typeof POSITION_NONE | typeof POSITION_TOP | typeo
                                         typeof POSITION_BOTTOM | typeof POSITION_LEFT;
 
 export interface OptionalProps {
+    // defaul tool
```
Comment (martin-badin) [inline types/react-svg-pan-zoom/index.d.ts:62]: fix typo

```suggestion
    // default tool
```

File: types/react-svg-pan-zoom/index.d.ts [hunk 2]
```diff
@@ -58,6 +59,9 @@ export type ToolbarPosition = typeof POSITION_NONE | typeof POSITION_TOP | typeo
...
+    defaultTool: string;
```
Comment (martin-badin) [inline types/react-svg-pan-zoom/index.d.ts:63]:
```suggestion
    defaultTool: 'none' | 'pan' | 'zoom-in' | 'zoom-out';
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/react-svg-pan-zoom/index.d.ts [hunk 3]
```diff
@@ -58,6 +59,9 @@ export type ToolbarPosition = typeof POSITION_NONE | typeof POSITION_TOP | typeo
                                         typeof POSITION_BOTTOM | typeof POSITION_LEFT;
 
 export interface OptionalProps {
+    // default tool
+    defaultTool: Tool;
```
Comment (martin-badin) [inline types/react-svg-pan-zoom/index.d.ts:63]: The type Tool includes extended 'auto' value but based on documentation 'defaultTool' doesn't include the value. You can use the type Tool without auto 
 ```suggestion
    defaultTool: Exclude<Tool, typeof TOOL_AUTO>;
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Review by martin-badin — APPROVED** [review]
Looks good. Thanks

**Comment by tvwenger** [comment]
Ready to merge

**Pull request merged**